### ISSUE TITLE
Error running 'testrunner': Cannot run 'testrunner' on '<default>' 

### ISSUE THREAD

**Issue by MiracleEEEE** [AlexeyDmitriev/JHelper#92]
i got "Error running 'testrunner': Cannot run 'testrunner' on '<default>' after i choosing the task in the select-box on the toolbar.

**Comment by AlexeyDmitriev** [comment]
Hi, what version of CLion do you use? Currently last supported version is 2017.2 unfortunately (I still can't find time to research and fix this)

**Comment by kilotaras** [comment]
Maybe this will help. clion logs

```
2018-06-23 11:28:47,635 [ 137175]  ERROR - penapi.actionSystem.impl.Utils - update failed for AnAction(name.admitriev.jhelper.actions.SwitchCMakeConfigurationAction) with ID=name.admitriev.jhelper.actions.SwitchCMakeConfigurationAction 
java.lang.NoSuchMethodError: com.jetbrains.cidr.cpp.execution.CMakeAppRunConfiguration.getBuildAndRunConfigurations()Lcom/jetbrains/cidr/cpp/execution/CMakeAppRunConfiguration$BuildAndRunConfigurations;
	at name.admitriev.jhelper.actions.SwitchCMakeConfigurationAction.getBuildConfigurationName(SwitchCMakeConfigurationAction.java:91)
	at name.admitriev.jhelper.actions.SwitchCMakeConfigurationAction.update(SwitchCMakeConfigurationAction.java:72)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performDumbAwareUpdate(ActionUtil.java:136)
	at com.intellij.openapi.actionSystem.impl.Utils.a(Utils.java:259)
	at com.intellij.openapi.actionSystem.impl.Utils.expandActionGroup(Utils.java:202)
	at com.intellij.openapi.actionSystem.impl.ActionToolbarImpl.a(ActionToolbarImpl.java:1050)
	at com.intellij.openapi.actionSystem.impl.ActionToolbarImpl.access$000(ActionToolbarImpl.java:52)
	at com.intellij.openapi.actionSystem.impl.ActionToolbarImpl$2.updateActionsImpl(ActionToolbarImpl.java:173)
	at com.intellij.openapi.actionSystem.impl.ToolbarUpdater$MyUpdateRunnable.run(ToolbarUpdater.java:186)
	at com.intellij.util.ui.UIUtil.invokeLaterIfNeeded(UIUtil.java:2886)
	at com.intellij.ide.IdeEventQueue.a(IdeEventQueue.java:173)
	at com.intellij.ide.IdeEventQueue.executeWhenAllFocusEventsLeftTheQueue(IdeEventQueue.java:132)
	at com.intellij.openapi.wm.impl.FocusManagerImpl.doWhenFocusSettlesDown(FocusManagerImpl.java:190)
	at com.intellij.openapi.actionSystem.impl.ToolbarUpdater.a(ToolbarUpdater.java:108)
	at com.intellij.openapi.actionSystem.impl.ToolbarUpdater.access$400(ToolbarUpdater.java:40)
	at com.intellij.openapi.actionSystem.impl.ToolbarUpdater$MyTimerListener.run(ToolbarUpdater.java:158)
	at com.intellij.openapi.actionSystem.impl.WeakTimerListener.run(WeakTimerListener.java:54)
	at com.intellij.openapi.actionSystem.impl.ActionManagerImpl$MyTimer.a(ActionManagerImpl.java:1439)
	at com.intellij.openapi.actionSystem.impl.ActionManagerImpl$MyTimer.a(ActionManagerImpl.java:1429)
	at com.intellij.openapi.actionSystem.impl.ActionManagerImpl$MyTimer.actionPerformed(ActionManagerImpl.java:1413)
	at javax.swing.Timer.fireActionPerformed(Timer.java:313)
	at javax.swing.Timer$DoPostEvent.run(Timer.java:245)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:762)
	at java.awt.EventQueue.access$500(EventQueue.java:98)
	at java.awt.EventQueue$3.run(EventQueue.java:715)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:732)
	at com.intellij.ide.IdeEventQueue.n(IdeEventQueue.java:786)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:727)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:395)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)

```

**Comment by whhone** [comment]
I encounter the same error on CLion 2018.1.

**Comment by AlexeyDmitriev** [comment]
Fixed in 0.15 (In CLion 2018.2) (The same code should probably work for 2017.3 and 2018.1 but I don't wanna check that since newer version exists).
